### ISSUE TITLE
Nothing happens when rendering a template

### ISSUE THREAD

**Issue by karmicdude** [Grokzen/docker-jinja#5]
```bash
$ python -V
Python 2.7.12

$ type python
python is hashed (/home/zengine/data/.environments/docker-jinja/bin/python)

$ pip freeze
blessings==1.6
bpython==0.16
curtsies==0.2.11
docker-jinja==0.1.0
docopt==0.6.2
greenlet==0.4.10
Jinja2==2.7.3
MarkupSafe==0.23
Pygments==2.1.3
PyYAML==3.11
requests==2.11.1
six==1.10.0
wcwidth==0.1.7
```
Template example:
```bash
$ cat Dockerfile.jinja 
FROM {{ OS }}
MAINTAINER {{ MAINTAINER }} 

ARG {{ ARG1 }}
```

Now i tried to run  dj:
```bash
$ dj --dockerfile Dockerfile.jinja --outfile Dockerfile --env OS=registry.local:5000/debin/debian:latest --env MAINTAINER=voronov.semyon@gmail.com --env ARG1=teststring
# or like this
$ dj -d Dockerfile.jinja -o Dockerfile -e OS=registry.local:5000/debin/debian:latest -e MAINTAINER=voronov.semyon@gmail.com -e ARG1=teststring
```

Final output: 
```bash
$ cat Dockerfile
FROM 
MAINTAINER  

ARG 
```

**Comment by Grokzen** [comment]
@z-engine Interesting problem. I will look over it during the weekend.

**Comment by Grokzen** [comment]
@z-engine Try the latest commit in master branch, the problem should be fixed there and a new release should be pushed to pypi very soon. I tested your exact case and i got it to work on my end now so you know.

**Comment by Grokzen** [comment]
@z-engine https://pypi.python.org/pypi/docker-jinja/1.0.0 Closing this as fixed as of version `1.0.0`
